### PR TITLE
Echo full openshift-tests command being used

### PR DESCRIPTION
**Pull request by jluhrsen**
Signed-off-by: Jamo Luhrsen <jluhrsen@gmail.com>

### CODE REVIEW

File: ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh [hunk 1]
```diff
@@ -94,7 +94,10 @@ if [[ -n "${TEST_SKIPS}" ]]; then
...
 fi
 
-openshift-tests "${TEST_COMMAND}" "${TEST_SUITE}" ${TEST_ARGS:-} \
+# using set -x so we will see the full openshift-tests command in the log. disable right after.
+set -x
+openshift-tests "${TEST_COMMAND}" "${TEST_SUITE}" "${TEST_ARGS:-}" \
     --provider "${TEST_PROVIDER}" \
     -o /tmp/artifacts/e2e.log \
     --junit-dir /tmp/artifacts/junit
+set +x
```
Comment (wking) [inline ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh:103]: do we need to turn it off after?  That's the end of the script.

File: ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh [hunk 2]
```diff
@@ -94,7 +94,10 @@ if [[ -n "${TEST_SKIPS}" ]]; then
     TEST_ARGS="${TEST_ARGS:-} --file /tmp/tests"
 fi
 
-openshift-tests "${TEST_COMMAND}" "${TEST_SUITE}" ${TEST_ARGS:-} \
+# using set -x so we will see the full openshift-tests command in the log. disable right after.
+set -x
+openshift-tests "${TEST_COMMAND}" "${TEST_SUITE}" "${TEST_ARGS:-}" \
```
Comment (wking) [inline ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh:99]: We can have multiple arguments in `TEST_ARGS`.  We can build it as a Bash array if we need to keep ShellCheck happy, or just leave master's unquoted expansion alone.

File: ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh (hunk 1 above)
Comment (jluhrsen) [reply ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh:103]: was just trying to keep xtrace off in the case that some future development happens at the end of this script.

File: ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh (hunk 2 above)
Comment (jluhrsen) [reply ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh:99]: my bad. the online shellcheck tool I was using flagged this so I blindly just quoted it. our validation doesn't seem to
care though so no tricks needed. just force pushed this PR to remove quotes from `TEST_ARGS`

File: ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh (hunk 1 above)
Comment (wking) [reply ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh:103]: there's no reason to ever put additional stuff at the end of this script.  If anything needs to run post-test, it should go in a separate step.

Comment (jluhrsen) [reply ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh:103]: ack

**Comment by jluhrsen** [comment]
@wking, good with this now?

**Comment by wking** [comment]
```console
$ curl -s https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/14183/rehearse-14183-pull-ci-openshift-installer-master-e2e-aws-shared-vpc/1336816247458762752/artifacts/e2e-aws-shared-vpc/openshift-e2e-test/container-logs/test.log | head -n2
secret/support created
+ openshift-tests run openshift/conformance/parallel --provider '{"type":"aws","region":"us-east-1","zone":"us-east-1b","multizone":true,"multimaster":true}' -o /tmp/artifacts/e2e.log --junit-dir /tmp/artifacts/junit
```

/lgtm

**Pull request merged**